NuGet.Config

```diff
@@ -10,11 +10,7 @@
     <add key="myget.org buildtools v3" value="https://www.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <add key="myget.org nugetbuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
     <add key="myget.org dotnet-core v3" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
-    <add key="myget.org buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
-    <add key="myget.org dotnet-corefx" value="https://www.myget.org/F/dotnet-corefx/" />
-    <add key="myget.org dotnet-corefxtestdata" value="https://www.myget.org/F/dotnet-corefxtestdata/" />
-    <add key="myget.org dotnet-core" value="https://www.myget.org/F/dotnet-core/" />
-    <add key="myget.org aspnet vnext" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
-    <add key="nuget.org v2" value="https://nuget.org/api/v2/" />
+    <add key="myget.org aspnet vnext v3" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
+    <add key="myget.org aspnet vnext" value="https://www.myget.org/F/aspnetvnext/" />
```

Comment:
Oh, I see--my question is wrong. It's actually "why leave the v2 reference here when it's not for any of the other sources?"